### DISCUSSION TITLE
Which learning rate schedular used by qwen 3 pretraining stages.

### DISCUSSION THREAD

**Discussion by VMD7** [QwenLM/Qwen3#1749] [General]
I have gone through the technical report but there is no mention of which learning rate schedular used. Do anyone know about what exactly they used in PT stages of qwen3.
